templates/editEvents.html

```diff
@@ -232,13 +232,13 @@ <h4 class="modal-title">
                 <div id="roundResultsListWrapper">
                   {{> roundResultsList roundId=_id advanceCount=advanceCount configurableAdvanceCount=true}}
                 </div>
-                <input type="number" min="1" max="{{competitorsInRound}}" name="advanceCount" class="form-control" value="{{advanceCount}}" />
+                <input type="number" min="1" max="{{participantsInRound}}" name="advanceCount" class="form-control" value="{{advanceCount}}" />
               </div>
             </div>
             <div class="modal-footer">
               <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
               <button type="submit" class="btn {{#if isSaveable}}btn-success{{/if}}" disabled="{{#unless isSaveable}}disabled{{/unless}}">
-                Advance competitors <span class="glyphicon glyphicon-step-forward"></span>
+                Advance participants <span class="glyphicon glyphicon-step-forward"></span>
```

Comment:
I think this is also another case of the UI showing participant instead of competitor?